ci/terraform/oidc/authorizer-orch-frontend.tf

```diff
@@ -0,0 +1,199 @@
+module "orch_frontend_authorizer_role" {
+  source      = "../modules/lambda-role"
+  environment = var.environment
+  role_name   = "orch-frontend-authorizer-role"
+  vpc_arn     = local.authentication_vpc_arn
+}
+
+locals {
+  authorizer_memory_size                 = lookup(var.performance_tuning, "authorizer", local.default_performance_parameters).memory
+  authorizer_provisioned_concurrency     = lookup(var.performance_tuning, "authorizer", local.default_performance_parameters).concurrency
+  authorizer_max_provisioned_concurrency = lookup(var.performance_tuning, "authorizer", local.default_performance_parameters).max_concurrency
+  authorizer_scaling_trigger             = lookup(var.performance_tuning, "authorizer", local.default_performance_parameters).scaling_trigger
+
+  alert_error_threshold      = 5
+  alert_error_rate_threshold = 10
+}
+
+resource "aws_lambda_function" "orch_frontend_authorizer" {
+  function_name = "${var.environment}-orch-frontend-authorizer-lambda"
+  role          = module.orch_frontend_authorizer_role.arn
+  handler       = "uk.gov.di.authentication.oidc.lambda.OrchFrontendAuthorizerHandler::handleRequest"
+  runtime       = "java17"
+
+  s3_bucket         = aws_s3_bucket.source_bucket.bucket
+  s3_key            = aws_s3_object.oidc_api_release_zip.key
+  s3_object_version = aws_s3_object.oidc_api_release_zip.version_id
+
+  code_signing_config_arn = local.lambda_code_signing_configuration_arn
+
+  publish     = true
+  timeout     = 900
```

Comment:
Where was this value taken from ? The previous authorizer was set to 30.

Reply:
Good spot have set to 30